GREEDY/View/MainScreen.cs

```diff
@@ -1,87 +1,50 @@
-﻿using GREEDY.Controllers;
-using System;
-using System.Collections.Generic;
+﻿using System;
 using System.Windows.Forms;
-using System.Configuration;
-using System.Data;
-using GREEDY.Models;
+using GREEDY.Services;
+using GREEDY.DataManagers;
 
-namespace GREEDY
+namespace GREEDY.View
 {
-    public partial class Greedy : Form
+    public partial class MainScreen : Form
     {
-        public Greedy()
+        private readonly IReceiptService _receiptService;
+        private readonly IImageGetter _photoImageGetter;
+        private readonly IImageGetter _fileImageGetter;
+
+        public MainScreen(ReceiptService receiptService)
```

Comment:
Should be IReceiptService and it would probably make sense to pass other dependencies as constructor arguments as well

Reply:
Yes, thanks